hugr/src/ops/custom.rs

```diff
@@ -28,6 +28,17 @@ pub enum ExternalOp {
     Opaque(OpaqueOp),
 }
 
+impl PartialEq for ExternalOp {
```

Comment:
was this needed for this PR? looks like the non-trivial comparisons don't have any coverage either

Reply:
I'll drop this, and move the discussion to the other PR
(https://github.com/CQCL/hugr/pull/923#discussion_r1562305666)